src/api/stops/StopsAPI.test.js

```diff
@@ -0,0 +1,56 @@
"country_code": "CH", "rank": 0.260139854137714, "translated_names": [], "mot": {"bus": false, "ferry": false, "gondola": false, "tram": false, "rail": true, "funicular": false, "cable_car": false, "subway": false}, "ident_source": "sbb", "id": "8507078", "code": "WBB", "ifopt": null},"geometry":{"type":"Point","coordinates":[7.445889,46.92866]}},{"type":"Feature","properties":{"uid": "28cf4f55b5c12c51", "name": "Bern Felsenau", "country_code": "CH", "rank": 0.273626372447381, "translated_names": [], "mot": {"bus": true, "ferry": false, "gondola": false, "tram": false, "rail": true, "funicular": false, "cable_car": false, "subway": false}, "ident_source": "sbb", "id": "8508051", "code": "BNFE", "ifopt": null},"geometry":{"type":"Point","coordinates":[7.444249,46.9685]}},{"type":"Feature","properties":{"uid": "8a5be589a20396ee", "name": "Bern Tiefenau", "country_code": "CH", "rank": 0.273626372447381, "translated_names": [], "mot": {"bus": true, "ferry": false, "gondola": false, "tram": false, "rail": true, "funicular": false, "cable_car": false, "subway": false}, "ident_source": "sbb", "id": "8508052", "code": "BNTI", "ifopt": null},"geometry":{"type":"Point","coordinates":[7.45019899999999,46.97274]}},{"type":"Feature","properties":{"uid": "3e6614fc41e84518", "name": "Ittigen bei Bern", "country_code": "CH", "rank": 0.27692307142111, "translated_names": [], "mot": {"bus": false, "ferry": false, "gondola": false, "tram": false, "rail": true, "funicular": false, "cable_car": false, "subway": false}, "ident_source": "sbb", "id": "8507069", "code": "ITT", "ifopt": null},"geometry":{"type":"Point","coordinates":[7.48412900000001,46.97331]}},{"type":"Feature","properties":{"uid": "58b64d5b6dead060", "name": "Bern Europaplatz", "country_code": "CH", "rank": 0.282352935350858, "translated_names": [], "mot": {"bus": false, "ferry": false, "gondola": false, "tram": false, "rail": true, "funicular": false, "cable_car": false, "subway": false}, "ident_source": "sbb", "id": "8504108", "code": "BNAS", "ifopt": null},"geometry":{"type":"Point","coordinates":[7.40611699999999,46.94421]}},{"type":"Feature","properties":{"uid": "aef478ea9a7cb78c", "name": "Bern  Europaplatz", "country_code": "CH", "rank": 0.282352935350858, "translated_names": [], "mot": {"bus": false, "ferry": false, "gondola": false, "tram": false, "rail": true, "funicular": false, "cable_car": false, "subway": false}, "ident_source": "sbb", "id": "8507082", "code": "BNAH", "ifopt": null},"geometry":{"type":"Point","coordinates":[7.40746899999999,46.94344]}},{"type":"Feature","properties":{"uid": "0998c82223ecac24", "name": "Bern St\u00f6ckacker", "country_code": "CH", "rank": 0.282352935350858, "translated_names": [], "mot": {"bus": false, "ferry": false, "gondola": false, "tram": false, "rail": true, "funicular": false, "cable_car": false, "subway": false}, "ident_source": "sbb", "id": "8504495", "code": "BNST", "ifopt": null},"geometry":{"type":"Point","coordinates":[7.401179,46.94635]}}]}',
```

Comment:
put this in a file that  you import in /src/setupTests.js

Reply:
@oterral Why should I add this file to the global setupTests.js file. I need this test data file only for this specific test. I could also import it in StopsAPI.test.js. What do you think?

Reply:
I have placed them in a new folder called data/ as some files are used in different tests.
And currently, I directly import the test data in the test file instead of in /src/setupTests.js.

Reply:
To answer you questions, something unreadable is not helpful for the next developer who wants to create/fix tests.
Here we have  2 choices:
   - create the most simplest FeatureCollection you need as response, with only properties you need for the test and with dummy coordinates [0,1],[0,2]
or
   -  move the content in a well formatted file in the data/ folder

Reply:
and I wanted you to move the import of  this complex FeatureCollection in setupTest.js because when a future layer class  will need to use this kind of  api response, a global object is already provided so it will be  easy to reuse.

Reply:
I have moved the test data file imports to setupTest.js.

Reply:
I have minimized the test data files. In some cases, you don't recognize a response anymore ;-)